R/ReportCard.R

```diff
@@ -60,6 +60,18 @@ ReportCard <- R6::R6Class( # nolint: object_name_linter.
       private$content <- append(private$content, TextBlock$new(text, style))
       invisible(self)
     },
+    #' @description Appends a `rmarkdown` R chunk to this `ReportCard`.
+    #'
+    #' @param text (`character(0)` or `character(1)`) the text
```

Comment:
Can we make it more explicit this is the code?

Reply:
See also comment regarding argument name.

Reply:
more than that there is no dot at the end of the line;o